codalab/lib/bundle_cli.py

```diff
@@ -313,7 +313,8 @@ def render_args(arguments):
                 if markdown:
                     name = HEADING_LEVEL_3 + name
                 return '%s%s:\n%s\n%s' % (
-                    ' ' * indent,
+                    # This is to make github Markdown format compatible with the readthedoc theme.
```

Comment:
Capitalize `GitHub` and should it be `Read the Docs`? https://docs.readthedocs.io/en/rel/theme.html

Reply:
Updated!